kuma/wiki/tests/test_views.py

```diff
@@ -409,11 +410,13 @@ def test_dont_render_previews_for_deferred_docs(self, mock_post):
 class DocumentSEOTests(UserTestCase, WikiTestCase):
     """Tests for the document seo logic"""
 
-    # NOTE(djf): In the past, we included the title of the "SEO Root"
-    # document in the title of every page, and this test tested for it.
-    # That feature has been removed now, and now this test verifies that
-    # the SEO root title does *not* appear in document titles.
-    def test_no_seo_title(self):
+    def test_get_seo_parent_doesnt_throw_404(self):
+        """bug 1190212"""
```

Comment:
The comment for [bug 1190212](https://bugzilla.mozilla.org/show_bug.cgi?id=1190212) is back 😢